translation_interface.go

```diff
@@ -18,4 +18,7 @@ type Translation interface {
 
 	// GetComponentMessages Get component messages
 	GetComponentMessages(name, version, locale, component string) (ComponentMsgs, error)
+
+	// GetComponentsMessages Get messages of multiple components
+	GetComponentsMessages(name, version string, locales, components []string) ([]ComponentMsgs, error)

```

Comment:
is it necessary to get more than one locale's transaltions?

Reply:
Yes. Because the server may serve clients with different locales.